scripts/jenkins-yoctobuild-init-script.sh

```diff
@@ -31,6 +31,12 @@ ln -sf ../go/bin/go /usr/local/bin/go
 ln -sf ../go/bin/godoc /usr/local/bin/godoc
 ln -sf ../go/bin/gofmt /usr/local/bin/gofmt
 
+# Download and install lzma using muslc.
+wget https://tukaani.org/xz/xz-5.2.4.tar.gz
+tar -xf xz-5.2.4.tar.gz
+mkdir /usr/local/musl
+(cd xz-5.2.4 && ./configure CC=musl-gcc --disable-shared --prefix=/usr/local/musl/ && make && sudo make install)
```

Comment:
I chose to install into /usr/local/musl somewhat arbitrary. It could as well be installed in $HOME/musl, or anywhere else in the users access path. I guess this boils down to, which do you prefer?